code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -84,7 +84,7 @@
 		to_chat(user, span_notice("You fill [src] with [trans] unit\s of the contents of [target]."))
 
 	else if(reagents.total_volume)
-		if(user.a_intent == INTENT_HARM)
+		if(user.a_intent == INTENT_HARM && !istype(target, /obj/item/hypospray))
```

Comment:
What is this meant to accomplish? If hyposprays should be fillable by beakers, they should be passing ``is_refillable`` and end here. If they aren't, then maybe the proper fix would be to make sure you can't splash contents onto things in your inventory?

Reply:
I didn't want to entirely remove the ability to splash reagents onto an item just to fix hyposprays, so I made an exception because they're the only item I'm aware of that you load with reagent containers

Reply:
That's the thing, they don't because hyposprays aren't directly refillable with reagents

Reply:
I see what you mean, but this is not the proper fix for it. There's also a var on reagent containers ``spillable`` that you can use to prevent them from spiling vials at all, since the intent for it seems to be to load it up in a hypospray only.

Reply:
That works too I guess, I wasn't intending to make hypospray vials never able to splash their contents on anything but it's not that big of a deal.